spec/controllers/request_controller_spec.rb

```diff
@@ -17,6 +20,11 @@
     expect(response).to render_template('list')
   end
 
+  it "renders the approximate number of requests" do
+    get :list, :view => 'all'
+    expect(response.body).to match(/About \d+ FOI requests found/)
```

Comment:
I'm not sure a controller spec is really appropriate here.

We're now calling `render_views` which will have a performance impact, and I don't think this is really testing any logic as such. In a controller spec we might want to check that `@results[:matches_estimated]` is set/accurate, but I don't really think that how we then interpolate that in to HTML is massively relevant here.

This would be better in a view spec if anything, but tbh, I'd just drop this.